src/async_fuse/memfs/cache/memory_cache/write_back_task.rs

```diff
@@ -54,24 +58,50 @@ fn calculate_period(
     }
 }
 
+/// Write a block back to the backend.
+async fn write_back_one_block<P, S>(
+    cache: Arc<MemoryCache<P, S>>,
+    ino: INum,
+    block_id: BlockId,
+    block: Block,
+    sender: oneshot::Sender<()>,
+) where
+    P: EvictPolicy<BlockCoordinate> + Send + Sync + 'static,
+    S: Storage + Send + Sync + 'static,
+{
+    cache.backend().store(ino, block_id, block).await;
+    sender.send(()).unwrap_or_else(|()| {
+        warn!("The receiver of pending block is closed unexpectedly.");
```

Comment:
The same as above.

Reply:
It's also enough. The sender here is to notify the caller (usually the FUSE operation handler) that this block is flushed to the backend. Although the Receiver held by the caller is dropped, the block has been flushed.

Reply:
This is enough. The receivers here are held by `MemoryCache`, if the receivers are closed unexpectedly, the `MemoryCache` must be dropped, and the async FUSE task and operation handlers must quit. It's not bad, because anyway, the blocks in write back task have been flushed to the backend normally.